PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalCheckoutRequestUnitTest.java

```diff
@@ -37,11 +38,13 @@ public void newPayPalCheckoutRequest_setsDefaultValues() {
         assertNull(request.getLandingPageType());
         assertNull(request.getBillingAgreementDescription());
         assertFalse(request.getShouldOfferPayLater());
+        assertFalse(request.getEnablePayPalAppSwitch());
+        assertNull(request.getUserAuthenticationEmail());
         assertFalse(request.getHasUserLocationConsent());
     }
 
     @Test
-    public void setsValuesCorrectly() {
+    public void setsValuesCorrectly(@TestParameter boolean appSwitchEnabled) {
```

Comment:
I'm not familiar with Robolectric's TestParameter. Will it run the test with a random value? Or run the test multiple times with different values?

Reply:
You'd usually have to provide which values to run.
In this case though, since the value is a boolean, it just runs for false and true.

Reply:
BTW, to be precise, `@TestParameter` is from the [TestParameterInjector](https://github.com/google/TestParameterInjector/) library.
We can't have two `@RunWith` classes; with 4.14-beta-1 robolectric now supports integration with `TestParameterInjector`, which IMO, has a better and concise way of doing parameterized tests. Robolectric used to do it differently.